apps/teamsjs-diagnostic-app/public/index.html

```diff
@@ -8,6 +8,7 @@
       href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css"
     />
     <title>Microsoft Teams Tab</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
```

Comment:
Out of curiosity, what are you using this third party stylesheet for?

Reply:
That book icon button that's in the top right corner of the API box (see picture above), I can change it though to be a general icon if it's not good to have that third party?

Reply:
I don't think it's a big deal, it's just taking a dependency on a resource (this CSS page) that we don't control. My *default* would be to only do that if we feel like it's giving us a particularly good benefit so that I wouldn't have to deal with bugs like "CSS file removed by owner"

Reply:
Ahhh i see, that makes sense